CONTRIBUTING.md

```diff
@@ -257,7 +257,7 @@ fdbuild --init-with-template kwinft-plasma-meta
 ```
 
 More information about this can be found in the
-[respective section](https://github.com/romangg/como/blob/lib-isolate/CONTRIBUTING.md#using-fdbuild)
+[respective section](https://github.com/winft/como/blob/master/CONTRIBUTING.md)
```

Comment:
That should still target the respective section "Using FDBuild", so the link would be:
https://github.com/winft/como/blob/master/CONTRIBUTING.md#using-fdbuild